turtle/build.gradle.kts

```diff
@@ -20,6 +20,11 @@ dependencies {
     testImplementation("io.mockk:mockk:1.12.5")
 }
 
+tasks.withType<Test>() {
+    useJUnitPlatform()
+    environment("LANG", "en_US.UTF8")
```

Comment:
without this change, I have the test `checkout_doNotCreateIfNecessary_givenBranchDoesNotExist` failing, because I get git's error message in french.